tests/Browser/Pages/HomePage.php

```
@@ -0,0 +1,36 @@
+    /**
+     * Get the URL for the page.
+     */
+    public function url(): string
+    {
+        return '/';
+    }
+
+    /**
+     * Assert that the browser is on the page.
+     */
+    public function assert(Browser $browser): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$browser'. (Unused Code Rules)

(UnusedFormalParameter)